kogito-kafka-quickstart-springboot/README.md

```diff
@@ -59,28 +118,66 @@ java -jar target/kogito-kafka-quickstart-sprintboot-{version}.jar
 
 To make use of this application it is as simple as putting a message on `travellers` topic with following content 
 
+* To examine ProcessedTravellers topic and verify upcoming messages will be processed
+
+```
+kafka_2.11-2.4.0/bin/kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic processedtravellers 
+```
+
+* Send Message to Topic
```

Comment:
`Send message that should be processed to Topic`

Reply:
fixed @radtriste